json/schemas/otp-contents-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "OTP Contents",
+    "description": "Defined contents of the OTP",
```

Comment:
"...of the RP2350 OTP" ?

Reply:
I'll go with "... of the RP-series device OTP"